### PR TITLE
Top 10 page

### PR DESCRIPTION
**Pull request by danquah**

### CODE REVIEW

File: controllers/stats-pre-renderer.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+'use strict';
+const _ = require('lodash');
+const kbhStatsApi = require('../services/kbh-billeder-stats-api');
+const auth0 = require('collections-online/lib/services/auth0');
+const Sevice = auth0.Service;
+const PLACEHOLDER = '{{STAT}}';
+
+const rendere = {
```
Comment (viktorshamal) [inline controllers/stats-pre-renderer.js]: missing an r

File: controllers/stats-pre-renderer.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    let motiftagsWeek = kbhStatsApi.motifTags('week');
+    let geotagsWeek = kbhStatsApi.geotags('week');
+    let motiftagsTotal = kbhStatsApi.motifTags();
+    let geotagsTotal = kbhStatsApi.geotags();
+    let usersWeek = kbhStatsApi.allUsersPoints('week');
+    let usersTotal = kbhStatsApi.allUsersPoints();
+
+    // Prepare a function that maps auth0 user_ids to human-readable names.
+    const mapUsers = async (data) => {
+      let management = await Sevice;
```
Comment (viktorshamal) [inline controllers/stats-pre-renderer.js]: missing an r

Comment (danquah) [reply controllers/stats-pre-renderer.js]: ..... where? :)

**Comment by danquah** [comment]
@viktorshamal like that?

**Pull request merged**